cloud-regionsrv-client.spec

```diff
@@ -16,7 +16,7 @@
 #
 
 
-%define base_version 9.3.1
+%define base_version 9.3.2
```

Comment:
No need for this bump, the main code has not changed and the sub-package has it's own version